applications/projects/runSofa/cmake/GeneratePluginConfig.cmake

```diff
@@ -8,6 +8,12 @@ macro(sofa_generate_plugin_config config_filename)
     get_property(_allTargetNames GLOBAL PROPERTY __GlobalTargetNameList__)
 
     list(LENGTH _allTargets nbTargets)
+
+    # if no plugin (or no deprecated "module") then quit
+    if (${nbTargets} EQUAL 0)
+        return()
```

Comment:
I'm worried when I see a `return` in a macro.
See this part of CMake doc

> A macro is executed as if the macro body were pasted in place of the calling statement. This has the consequence that a [return()](https://cmake.org/cmake/help/latest/command/return.html#command:return) in a macro body does not just terminate execution of the macro; rather, control is returned from the scope of the macro call. To avoid confusion, it is recommended to avoid [return()](https://cmake.org/cmake/help/latest/command/return.html#command:return) in macros altogether.

Reply:
👍